apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -215,7 +215,7 @@ export const renderVerifyUnwrapped = ({
 }) => (
   <AnimationWrapper>
     <DeviceBlocker />
-    <Animation animation={getDeviceAnimation(modelId, type, "verify")} />
+    <Animation small animation={getDeviceAnimation(modelId, type, "verify")} />
```

Comment:
I guess it's a fine solution, as we don't really have a responsive design system

Reply:
Temporary, as everything